src/agent_assist/agent_assist.py

```diff
@@ -0,0 +1,557 @@
+        """
+        client = services.participants.ParticipantsClient(
+            credentials = self.creds
+            )
+
+        request = types.GetParticipantRequest(
+                    name=participant_name,
+                )
+
+        response = client.get_participant(request=request)
+
+        return response
+
+    def update_participant(
+        self,
+        participant_name: str = None,
+        obj: types.Participant = None,
+        **kwargs
+    ) -> types.Participant:
+        """
+        Updates the participant object based on the provided args.
+        Args:
+            participant_name: str
+                Full name of the participant to be updated.
+            obj:
+                The participant object to be updated.
+        """
+        if obj:
+            participant = obj
+            participant.name = participant_name
+
+        else:
+            if not participant_name:
+                raise ValueError(
+                    "Atleast one of participant object or participant "
+                    "name is required to make the update."
+                )
+
+            participant = self.get_participant(
+                participant_name = participant_name
+                )
+
+        # set participant attributes from kwargs
+        for key, value in kwargs.items():
+            setattr(participant, key, value)
+
+        paths = kwargs.keys()
+        mask = field_mask_pb2.FieldMask(paths=paths)
+
+        client = services.participants.ParticipantsClient(
+            credentials = self.creds
+            )
+
+        request = types.UpdateParticipantRequest()
+
+        request.participant = participant
+        request.update_mask = mask
+
+        response = client.update_participant(request = request)
+
+        return response
+
+    def analyze_content_text(
+        self,
+        text,
+        participant,
+        language_code = "en-US"
+        ) -> types.AnalyzeContentResponse:
+        """Analyze text message content from a participant.
+
+        Args:
+            text (str):
+                The input text string that needs to be analyzed.
+            participant (str):
+                The participant id of the participant for the
+                corresponding text utterance being analyzed.
+            language_code (str):
+                Language of the inputted text.
+
+        Returns:
+            Response message from the analyze content.
+            type: google.cloud.dialogflow.v2beta1.Participants.AnalyzeContent
+        """
+
+        client = services.participants.ParticipantsClient(
+            credentials= self.creds
+            )
+        text_input = {"text": text, "language_code": language_code}
+
+        logging.info("Running analyze content...")
+        response = client.analyze_content(
+            participant = participant,
+            text_input = text_input
+            )
+        logging.info(f"analyze content response: {response}")
```

Comment:
Remove logging `response` or set to DEBUG

Reply:
done